docs/CSSandSCSSBestPractices.md

```diff
@@ -0,0 +1,160 @@
+# vets.gov CSS/SCSS Best Practices (Draft)
+
+(_Created: April 28, 2017_)
+_Last Updated: May 3, 2017_
+
+
+## Overview
+
+This is high-level view of the current (S)CSS architecture, and a set of guidelines for writing scalable, maintainable, performant CSS and SCSS.
+
+## Background
+
+As Vets.gov has grown to encompass more features and products, our CSS has grown with it. The beta version of the site used the Foundation CSS framework, with custom CSS added as needed. Subsequent versions of the site shifted to the [United States Web Design Standards](https://standards.usa.gov) (USWDS) framework. 
+
+Portions of the site still use the Foundation grid. Newer templates use the USWDS grid. Most of the site's components use VA-specific CSS.
+
+## Current dependencies
+
+Vets.gov uses Sass as its preprocessor with SCSS syntax.
+
+- USWDS 0.10.0
+    - normalize.css
+    - Bourbon 
+    - Neat
+- Foundation (deprecated; please do not use)
+- FontAwesome
+
+## Browser support
+
+| Browser | Minimum version  |
```

Comment:
Maybe link to https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Work%20Practices/Engineering/DocumentedDecisions/Browser%20Support.md